### PR TITLE
January 2024 website overhaul

### PR DESCRIPTION
**Pull request by duongand**
Items to review:

- [x] Year of the Dragon announcement image and date
- [x] Google Font link in .html <head> changed to DM Sans
- [x] Moved left panel items into a container
- [x] CSS changes to announcements to appropriately take space within DOM to position correctly relative to one another
- [x] Minor text/description changes in content
- [x] CSS changes to locations to position in top half of the right panel
- [ ] Worked overflow issues within content element
- [x] CSS changes to announcement images
- [x] Updated left announcement image and text for Lunar New Years
- [x] CSS changes to content separator
- [x] favicon added (Dharma Wheel)

Be sure to pull changes locally to observe changes to actual website by:

1. git checkout home-styling
2. git pull origin home-styling
3. Go live on vscode

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -7,6 +7,13 @@
     <link href="styles.css" rel="stylesheet" />
     <meta name="author" content="" />
     <meta name="description" content="" />
+    <link rel="icon" type="image/x-icon" href="assets/dharma-wheel.png" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=DM+Sans:opsz@9..40&display=swap"
```
Comment (noraacode) [inline index.html:14]: So this applies dm sans to the html? Why does it have to be from Google API

File: index.html [hunk 2]
```diff
@@ -16,123 +23,140 @@
...
+          <div class="announcements-left">
+            <img
+              class="announcement-images"
+              src="assets/lunar-dragon.jpg"
+              alt="Lunar New Years Animal - Dragon"
+              height="315px"
+              width="430px"
+            />
+            <div>Lunar New Years - Year of the Dragon</div>
+            <div>February 10, 2024</div>
```
Comment (noraacode) [inline index.html:80]: Thay said it's celebrated the 18th I believe

File: index.html [hunk 3]
```diff
@@ -7,6 +7,13 @@
     <link href="styles.css" rel="stylesheet" />
     <meta name="author" content="" />
     <meta name="description" content="" />
+    <link rel="icon" type="image/x-icon" href="assets/dharma-wheel.png" />
```
Comment (noraacode) [inline index.html:10]: This icon is nice. :0

File: styles.css [hunk 4]
```diff
@@ -2,11 +2,11 @@ body {
   margin: 0px;
   box-sizing: border-box;
   background-color: #e9e3c8;
+  font-family: 'DM Sans', sans-serif;
```
Comment (noraacode) [inline styles.css:5]: Oh wait it's right here. Oh yes, then why D:

File: index.html [hunk 5]
```diff
@@ -16,123 +23,140 @@
...
-        <div>
-          <span class="location-title">Chùa Phật Giác</span></br>
-          10022 Gaines Road</br>
-          Sugar Land, TX 77498
-        </div>
-        <div>
-          <span class="location-title">Chùa Phật Tâm</span></br>
-          6310 Reega Ave.</br>
-          Egg Harbor Twp, NJ 08234
+        <div class="locations-container">
```
Comment (noraacode) [inline index.html:140]: Wait, why are all the locations smushed to the top now... Last time it was spread out... just a design comment

**Review by noraacode — APPROVED** [review]
I made some comments but none towards the code. Maybe the most code-related is the comment on line 137 in HTML

File: index.html (hunk 1 above)
Comment (duongand) [reply index.html:14]: Easiest method to import font into HTML project

File: index.html (hunk 5 above)
Comment (duongand) [reply index.html:140]: I think it looks better, but we can change the height of right panel

File: index.html (hunk 2 above)
Comment (duongand) [reply index.html:80]: Please review commit ID, [91b92fe](https://github.com/Tu-Vien-Van-Phat-Danh/tu-vien-van-phat-danh.github.io/pull/6/commits/91b92feca30c5d51c0e6c5bbf3b3106a879cec25).

Comment (noraacode) [reply index.html:80]: LGTM

**Pull request merged**